backend/data/src/main/kotlin/io/tolgee/formats/android/out/AndroidStringsXmlFileWriter.kt

```diff
@@ -67,11 +66,13 @@ class AndroidStringsXmlFileWriter(private val model: AndroidStringsXmlModel) {
       this.textContent = contentToAppend.text
     }
 
-    if (contentToAppend.children != null) {
-      contentToAppend.children.forEach {
-        val imported = this.ownerDocument.importNode(it, true)
-        this.appendChild(imported)
-      }
+    contentToAppend.children?.forEach {
+      val imported = this.ownerDocument.importNode(it, true)
+      this.appendChild(imported)
     }
   }
+
+  private fun Element.optionalComment(comment: String?) {
+    comment?.let { comment(" $it ") }
```

Comment:
I prefer how the comments look when there are spaces around them. Is it ok to do this?

Reply:
Yes